ParisBNF/abb/BNFabb126.xml

```diff
@@ -0,0 +1,309 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BNFabb126">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Life of Shenoute, life of Pishoy, life of Gabra Manfas Qǝddus</title>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p>
+                                            This file is licensed under the Creative Commons
+                                            Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10087868n">BnF Éthiopien d'Abbadie 126</idno>
+               </msIdentifier>
+<physDesc>
+<objectDesc form="Codex">
+<supportDesc>
+  <support>
+     <material key="parchment"/>
+  </support>
+  <extent>
+    <measure unit="leaf">136</measure>
+    <dimensions type="outer" unit="mm">
+       <height>285</height>
+       <width>210</width>
+    </dimensions>
+  </extent>
+</supportDesc>
+</objectDesc>
+
+<additions>
+<list>
+
+<item xml:id="e1">
+<locus target="1r"/>
+<desc type="OwnershipNote"></desc>
+<q xml:lang="gez">ዝመጽሐፍ፡ ዘእንጦንዮስ፡ ዘአበዲየ፡ ዘአጥረዮ፡ በንዋዩ።</q>
+</item>
+
+<item xml:id="e2">
+<locus target="1r"/>
+<desc>Signature of <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName> and library seal.</desc>
+</item>
+
+</list>
+</additions>
+
+</physDesc>
+
+<history>
+<provenance>The manuscript comes from <placeName ref="LOC2422DagaEs">Dāgā ʾƎsṭifānos</placeName>.</provenance>
+</history>
+
+<additional>
+    <adminInfo>
+        <recordHist>
+            <source>
+<listBibl type="catalogue">
+           <bibl>
+              <ptr target="bm:Abbadie1859Cat"/>
+              <citedRange unit="page">145-146</citedRange>
+              <citedRange unit="number">126</citedRange>
+           </bibl>
+
+           <bibl>
+              <ptr target="bm:Chaine1912Catalogue"/>
```

Comment:
You can add a note in p2 to indicate that the manuscript was used by P. Marrassini in his edition of the Vita of GMQ.